dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/ContainerHolder.java

```diff
@@ -0,0 +1,51 @@
+package com.dotmarketing.portlets.templates.design.bean;
+
+import com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * it is block that can contain {@link com.dotmarketing.portlets.containers.model.Container}
+ */
+public class ContainerHolder {
+
+    private boolean preview;
+    private List<String> containers;
+
+    @JsonCreator
+    public ContainerHolder(@JsonProperty("containers")  final List<String> containers) {
+        this.containers = containers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "containers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)